### PR TITLE
fix duplicate args for collector_encryption

### PR DESCRIPTION
**Pull request by vetinari**
this may happen if someone already uses collector_encryption...

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

**Comment by vetinari** [comment]
see also https://github.com/zalando-zmon/opentracing-utils/commit/24821cab969cab6c8ef840a8977c60a4857ac15f#r43044567

**Comment by vetinari** [comment]
:+1:

File: opentracing_utils/tracers.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ def init_opentracing_tracer(tracer, **kwargs):
         collector_encryption = "tls"
         if scheme == "http":
             collector_encryption = "none"
+        if 'collector_encryption' in kwargs:
```
Comment (mohabusama) [inline opentracing_utils/tracers.py:54]: hmm, I would expect that we give kwargs higher precedence.

Comment (vetinari) [reply opentracing_utils/tracers.py:54]: we do?

Comment (mohabusama) [reply opentracing_utils/tracers.py:54]: Well, if the caller passes `collector_encryption` then I would assume they know what they are doing, and implicitly changing that value by the lib could be confusing for them.

Comment (vetinari) [reply opentracing_utils/tracers.py:54]: isn't that what `collector_encryption = kwargs.pop('collector_encryption', collector_encryption)` does?

Comment (mohabusama) [reply opentracing_utils/tracers.py:54]: Ah right 🤦 !

**Comment by mohabusama** [comment]
👍

**Pull request merged**